src/cljs/main/broadfcui/page/workspace/method_configs/launch_analysis.cljs

```diff
@@ -60,14 +60,13 @@
                     (cond (= (entity->id (first row-data)) (:selected-entity @state)) "yellow"
                       (even? row-index) (:background-light style/colors)
                       :else "#fff")})
-      :entity-name-renderer (fn [e]
-                              (let [entity-id (entity->id e)]
-                                (style/create-link
-                                  {:text (:name entity-id)
-                                   :onClick #(swap! state assoc
-                                                    :selected-entity entity-id
-                                                    :workflow-count (common/count-workflows
-                                                                      e (:root-entity-type props)))})))}]]
+      :on-row-click (fn [row-index row-data]
+                      (let [row (first row-data)
+                            entity-id (entity->id row)]
```

Comment:
Rename `row` to `entity`, and `row-data` to `row`.  You can also rename `row-index` to `_` to indicate it's unused (and it gets rid of an editor warning for me at least).